### ISSUE TITLE
NTR: 3-methyladipic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8640]
New term request 3-methyladipic acid high in urine (elthat_190101150534):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000466
- label: Elevated urine 3-methyladipic acid level
- parents: [Dicarboxylic aciduria (HP:0003215)](https://hpo.jax.org/app/browse/term/HP:0003215)
- pmid: PMID:29909962
- def: The amount of 3-methyladipic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8640
- synonyms:
